github-pullrequest-plugin/src/main/java/org/jenkinsci/plugins/github/pullrequest/trigger/JobRunnerForCause.java

```diff
@@ -137,8 +139,22 @@ private static boolean cancelQueuedBuildByPrNumber(final int id) {
                 NUMBER.param(String.valueOf(cause.getNumber()))
         ));
         GitHubPRBadgeAction gitHubPRBadgeAction = new GitHubPRBadgeAction(cause);
-        //TODO no way to get quietPeriod, so temporary ignore it
-        return asParameterizedJobMixIn(job).scheduleBuild2(0, new CauseAction(cause), new ParametersAction(values),
+
+        ParameterizedJobMixIn parameterizedJobMixIn = asParameterizedJobMixIn(job);
+
+        int quietPeriod = 0;
+
+        // TODO: figure out a better way to do this
+        try {
+            Object mixinJob = FieldUtils.readField(parameterizedJobMixIn, "val$job", true);
```

Comment:
yep, that was the only field that [FieldList.getAllFields](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/reflect/FieldUtils.html#getAllFields(java.lang.Class)) returned.
And getting all available methods was not helpful either.